app/(editor)/editor/page.tsx

```diff
@@ -6,8 +6,7 @@ import { components as Components } from '@/components/StoryblokProvider';
 import { resolveRelations } from '@/utilities/resolveRelations';
 import { notFound } from 'next/navigation';
 import ComponentNotFound from '@/components/Storyblok/ComponentNotFound';
-
-const activeEnv = process.env.NODE_ENV || 'development';
```

Comment:
I got rid of this check. If the content is coming through the `(storyblok)` route segment assume only published content and vice versa for the `(editor)` route group. This seems like a safe assumption to me but let me know if you think otherwise.

Reply:
What if we're previewing locally but not in the visual editor iFrame? Eg, sometimes I like to go directly to localhost:3000 in my browser.

Reply:
That is a fair use case. I'll restore it.